UmbracoCms.8.1.1/App_Data/Models/Athlete.generated.cs

```diff
@@ -22,7 +22,7 @@ namespace Umbraco.Web.PublishedModels
 {
 	/// <summary>Athlete</summary>
 	[PublishedModel("athlete")]
-	public partial class Athlete : PublishedContentModel
+	public partial class Athlete : PublishedContentModel, IContentBase, INavigationBase
```

Comment:
Did you have to edit this by hand or did it just happen?
If you had to edit it by hand I would suggest you make a new class that inherits this one and add the extra interfaces, otherwise someone will have to keep making these manual edits each time the class is generated that will be tedious and error prone.

Reply:
No it was all generated